group1/...

```diff
@@ -0,0 +1,3 @@
+M	group1/comm/static/comm_tool.css
+M	group1/comm/static/comm_tool.js
+M	group1/comm/templates/comm/index.html
```

Comment:
@deltin00 looks like you inadvertantly created & added a file called "...". can you remove this from the PR ? then you are G2G.